test/governance/utils/Votes.test.js

```diff
@@ -21,41 +27,56 @@ contract('Votes', function (accounts) {
 
   describe('performs voting operations', function () {
     beforeEach(async function () {
-      this.tx1 = await this.votes.mint(account1, 1);
-      this.tx2 = await this.votes.mint(account2, 1);
-      this.tx3 = await this.votes.mint(account3, 1);
+      this.txs = await Promise.all(Object.entries(amounts).map(kv => this.votes.mint(...kv)));
```

Comment:
I feel that we should avoid this pattern because it hurts readability, which is important for tests as made explicit [in the new guidelines](https://github.com/frangio/openzeppelin-contracts/blob/5533d7e9fcba862797231c5961ea452a12ba3083/GUIDELINES.md#testing).

My recommendation here would be to use a for loop, sort of like this:

```suggestion
      this.txs = [];
      for (const [account, amount] of Object.entries(amounts)) {
        this.txs.push(await this.votes.mint(account, amount));
      }
```

It doesn't have to be exactly like this but I'm just hoping we can make it a bit more readable with simpler code.